test/Microsoft.VisualStudio.Sdk.TestFramework.Tests/SdkTestUtilitiesTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#if WINDOWS || NETFRAMEWORK
+
+using Microsoft.VisualStudio.Sdk.TestFramework.Tests;
+
+[Collection(MockedVS.Collection)]
+public class SdkTestUtilitiesTests : LoggingTestBase
+{
+    public SdkTestUtilitiesTests(GlobalServiceProvider sp, ITestOutputHelper logger)
+        : base(logger)
+    {
+        sp.Reset();
+    }
+
+    [Fact]
+    public async Task LoadPackageAsync()
+    {
+        SomePackage package = await SdkTestUtilities.LoadPackageAsync<SomePackage>(this.Logger.WriteLine);
+        Assert.NotNull(package);
```

Comment:
a further test here would be asking the package for a service it doesn't provide (but exists globally), that would ensure the GetService is calling through to the parent provider (the one given to it during SetSite) which proves its 'plugged in' to the overall service architecture

Reply:
Good idea.